### ISSUE TITLE
Incorrect handling of pressure unit in HEMCO standalone

### ISSUE THREAD

**Issue by yuanjianz** [geoschem/HEMCO#277]
### Your name

Yuanjian Zhang

### Your affiliation

WashU

## Issue
HEMCO standalone, different from HEMCO within geos-chem, has a internal pressure unit of Pa instead of hPa 
(See [geos-chem/GeosCore/hco_interface_gc_mod.F90](https://github.com/geoschem/geos-chem/blob/1ef3d7f88da95ca2425267fc4f96bd1524fc985f/GeosCore/hco_interface_gc_mod.F90#L3326))

Processed meteorological fields for MERRA-2 and GEOS-FP are separately in Pa and hPa.

Unlike geos-chem, HEMCO standalone does not have the automatic met-specific unit conversion in 
[geos-chem/GeosCore/flexgrid_read_mod.F90](https://github.com/geoschem/geos-chem/blob/1ef3d7f88da95ca2425267fc4f96bd1524fc985f/GeosCore/flexgrid_read_mod.F90#L1284-L1289)

So pressure is mistakenly treated as Pa if using GEOS-FP(and GEOS-IT) to drive HEMCO standalone. Possibly making output emission field in kg/m2/s 1000 times smaller.

Related issues: #275

## Workaround
Manually use a `hPa2Pa` scale factor of 100.0 in configuration.

But I also wish this can be addressed in a more natural manner like in GC-Classic and GCHP.

### What are the steps to reproduce the bug?

use standard run directory creation script to run a HEMCO standalone instance with GEOS-FP

### What HEMCO version were you using?

standalone

### What environment were you running HEMCO on?

Local cluster

### What compiler and version were you using?

gcc 10.2.0

### Will you be addressing this bug yourself?

Yes, but I will need some help

### In what configuration were you running HEMCO?

Standalone

### As what resolution were you running HEMCO?

all

### What meterology fields did you use?

Other (please explain in additional information section below)

### Additional information

GEOS-FP and GEOS-IT

**Comment by lizziel** [comment]
Thanks @yuanjianz for reporting this. I am curious if this means all of the offline emissions for GEOS-FP are incorrect since they are generated by using HEMCO standalone with the processed meteorology.

@yantosca, @msulprizio - do we know who generated the FP offline emissions?

**Comment by yuanjianz** [comment]
Hi @lizziel, I happen to have the codes for FP offline emissions. It seems BioVOC, soilNOx and seasalt are independent of pressure. Only dust is affected:
https://github.com/geoschem/HEMCO/blob/3bd40ad84c46c36107d83b5353e57ce279b593f0/src/Extensions/hcox_dustdead_mod.F#L420-L421
I checked the dust offline emission codes. They are quite old and labeled v11.01 and the hemco_stanalone.x needs to be compiled together with GC-classic. In its HEMCO_sa_Config:
```
* PSFC /ExtData/GEOS_0.25x0.3125/GEOS_FP/$YYYY/$MM/GEOSFP.$YYYY$MM$DD.I3.025x03125.nc PS    $YYYY/1-12/1-31/0-23 C xy  hPa     * - 1 1
```
I check GEOS-FP dust emission and they seem correct. I am not quite sure if it is because old HEMCO is compiled with GC-Classic and it has the automatic unit conversion.

**Comment by yuanjianz** [comment]
I am opening this issue since I am updating our ancient offline emissions code to HEMCO v3.9.0.

**Comment by yantosca** [comment]
We can now close this issue as the corresponding PR #284 has been merged into the HEMCO 3.10.0 development stream.

@yuanjianz: Please make sure to use this fix when recomputing the offline emissions.
